index.html

```diff
@@ -17,59 +17,120 @@
+                    <h3 class="work-name">Multi-Post Stories</h3>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="work-skills grid-container">
+                        <li class="gird">CSS</li>
+                        <li class="gird">HTML</li>
+                        <li class="gird">Bootstrap</li>
+                        <li class="grid0">Ruby</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
             </div>
         </section>
-        <hr class="break-skill">
-       <section class="contact">
-           <ul class="contact-container">
-                <li>
-                    <a href="#"><img src="img/github.png" alt="github" class="github"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/linkin.png" alt="linkin" class="linkin"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/gesture.png" alt="gesture" class="gesture"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/twitter.png" alt="twitter" class="twitter"></a>
-                </li>
-                <li>
-                    <a href="#"><img src="img/m.png" alt="m" class="m"></a>
-                </li>
-           </ul>
+        <section class="about-section">
+            <h2>About Me</h2>
+            <hr class="break-about">
+            <p class="description"> I am good at the following web technologies and skills. If you have any questions about web development, don't hesitate to contact me through the links below</p>
+            <div class="skill skill-container">
+                <div class="language">
+                    <h3>Language</h3>
+                    <ul class="lan-container">
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="framework">
+                    <h3>Framework</h3>
+                    <ul class="fra-container">
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                        <li>Bootstrap</li>
+                    </ul>
+                </div>
+                <div class="tools">
+                    <h3>Tools</h3>
+                    <ul class="to-container">
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
```

Comment:
**Changes Required**
- [ ] The `ul` looks good. Just increase the line height of the `li` items to look like that of the figma design. Following designs will definitely be one of your roles as a front-end engineer.